models/index.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+
+const fs = require("fs");
+const path = require("path");
+const Sequelize = require("sequelize");
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || "development";
+const config = require(__dirname + "/../config/config.js")[env];
+const db = {};
+
+let sequelize;
```

Comment:
can we move all the sequelize logic in a function in a file called sequelize.js that we can call in here?